lib/waterline/query/dql/update.js

```diff
@@ -137,6 +137,7 @@ function createBelongsTo(valuesObject, cb) {
       var pk = val[model.primaryKey];
 
       valuesObject.values[item] = pk;
+      _.remove(valuesObject.associations.models, function(_item) { return _item == item; });
```

Comment:
`delete valuesObject.associations.models[item];` won't work causes .models is array of string